src/librustc_data_structures/small_vec.rs

```diff
@@ -234,8 +43,8 @@ mod tests {
         let v: OneVector<isize> = OneVector::new();
         assert_eq!(0, v.len());
```

Comment:
Any of the functionality of smallvec itself shouldn't be tested here I think

Reply:
I guess we sould keep tests for the `expect_one()` function, right? That's the local function that isn't present in smallvec and we implement locally. Also, shall we keep or remove the benchmarks?

Reply:
Remove benchmarks, keep the expect_one test(s).